mne/io/edf/edf.py

```diff
@@ -78,47 +79,47 @@ def find_edf_events(raw):
 
 
 class RawEDF(BaseRaw):
-    """Raw object from EDF, EDF+, BDF file.
+    """Raw object from EDF, EDF+ or BDF file.
 
     Parameters
     ----------
     input_fname : str
-        Path to the EDF+,BDF file.
+        Path to the EDF, EDF+ or BDF file.
     montage : str | None | instance of Montage
-        Path or instance of montage containing electrode positions.
-        If None, sensor locations are (0,0,0). See the documentation of
+        Path or instance of montage containing electrode positions. If None,
+        sensor locations are (0,0,0). See the documentation of
         :func:`mne.channels.read_montage` for more information.
     eog : list or tuple
-        Names of channels or list of indices that should be designated
-        EOG channels. Values should correspond to the electrodes in the
-        edf file. Default is None.
+        Names of channels or list of indices that should be designated EOG
+        channels. Values should correspond to the electrodes in the file.
+        Default is None.
     misc : list or tuple
-        Names of channels or list of indices that should be designated
-        MISC channels. Values should correspond to the electrodes in the
-        edf file. Default is None.
+        Names of channels or list of indices that should be designated MISC
+        channels. Values should correspond to the electrodes in the file.
+        Default is None.
     stim_channel : 'auto' | str | list of str | int | list of int
-        It defaults to 'auto' where channels named 'status' or 'trigger'
-        are set as 'stim'. When str (or list of str) channels matching this
-        string are set as 'sitm'. The matching is not case sensitive, same for
-        the default behavior. When int (or list of ints) the channel
-        corresponding to this position is set as channels of type 'stim'.
-
-        .. warning:: 0.18 does not allow for stim channel synthesis from
-                     the TAL channels called 'EDF Annotations' or
-                     'BDF Annotations'. The TAL channel is parsed
-                     and put in the raw.annotations attribute.
-                     Use :func:`mne.events_from_annotations` to obtain
-                     events for the annotations instead.
+        Defaults to 'auto', which means that the type of channels named
+        'status' or 'trigger' (case insensitive) is set to STIM. If str (or
+        list of str), the type of channels matching the name(s) is set to STIM.
```

Comment:
`the type of channels matching the name(s) is` --> `the channels matching the name(s) are`